app/routes/titiler/algorithms/dist_alerts.py

```diff
@@ -21,3 +26,41 @@ class DISTAlerts(Alerts):
     )
 
     record_start_date: str = "2020-12-31"
+
+    tree_cover_density_mask: Optional[int] = None
+    tree_cover_density_data: Optional[ImageData] = None
+
+    tree_cover_height_mask: Optional[int] = None
+    tree_cover_height_data: Optional[ImageData] = None
+
+    # the highest loss year that is used to exclude alerts for
+    # the purpose of showing only alerts in forests
```

Comment:
Maybe in a following change you can just add the extra sentence "Should be 2021 or later, since 2020 tree height data will cover tree loss before 2021.".

Reply:
Will do. I have already added a validator to enforce that in the endpoint itself here too: https://github.com/wri/gfw-tile-cache/blob/gtc-3025/filter-alerts-by-forest/app/routes/titiler/umd_glad_dist_alerts.py#L65